src/motors.cpp

```diff
@@ -34,10 +34,21 @@ namespace bzzz
     void MotorDriver::arm(void)
     {
         m_armStatus = 1;
+        m_frontLeftEsc.writeMicroseconds(ZERO_ROTOR_SPEED);  // start the motors. sets the ESC speed to IDLE_ROTOR_SPEED
```

Comment:
Why are you setting `ZERO_ROTOR_SPEED` then `ARM_ROTOR_SPEED` and then `ZERO_ROTOR_SPEED` again?

To simplify the code, you can implement a private method called `writeAll` that will look something like

```c++
void Servo::writeAll(int commonMicros) {
    m_frontLeftEsc.writeMicroseconds(commonMicros); 
    m_frontRightEsc.writeMicroseconds(commonMicros);
    m_backLeftEsc.writeMicroseconds(commonMicros);
    m_backRightEsc.writeMicroseconds(commonMicros);
}
```